com.cerebralnomad.recipescribe.yml

```diff
@@ -0,0 +1,77 @@
+id: com.cerebralnomad.recipescribe
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: recipe_scribe-2.0.1.py
+
+finish-args:
+  - --filesystem=home:rw
+  - --socket=fallback-x11
+  - --socket=wayland
```

Comment:
are you sure this work with wayland?

Reply:
The binary runs fine on wayland, but I just tested it and the flatpak does not. It didn't occur to me that it might be a problem. 
I'll have to dig into that and see what the issue is.
In the meantime I guess I'll have to use socket=x11 instead